components/select/src/auro-select.js

```diff
@@ -611,46 +626,55 @@ export class AuroSelect extends AuroElement {
   configureSelect() {
 
     this.addEventListener('keydown', (evt) => {
-      if (evt.key === 'ArrowUp') {
-        evt.preventDefault();
+      if (this.shadowRoot.activeElement === this.dropdown) {
```

Comment:
Why did we have to add this line? The select is already the activeelement or you wouldn't pick up the keydown event.